go.mod

```diff
@@ -3,8 +3,7 @@ module github.com/pingcap/tidb
 go 1.16
 
 require (
-	cloud.google.com/go v0.54.0 // indirect
-	cloud.google.com/go/storage v1.6.0
+	cloud.google.com/go/storage v1.16.1-0.20210826124857-08169baa564d
```

Comment:
perhaps you need to keep the comment about the failing br_gcs test, until googleapis/google-cloud-go#4609 is released.

Reply:
v1.16.1 released

Reply:
https://github.com/googleapis/google-cloud-go/releases/tag/storage/v1.16.1